libs/zzz/formula/src/data/char/sheets/Soldier0Anby.ts

```diff
@@ -1,59 +1,101 @@
-import { cmpGE, prod, sum } from '@genshin-optimizer/pando/engine'
+import { cmpGE, prod, subscript, sum } from '@genshin-optimizer/pando/engine'
 import { type CharacterKey } from '@genshin-optimizer/zzz/consts'
 import { allStats, mappedStats } from '@genshin-optimizer/zzz/stats'
 import {
   allBoolConditionals,
-  allListConditionals,
-  allNumConditionals,
-  enemyDebuff,
+  customDmg,
   own,
   ownBuff,
   percent,
   register,
   registerBuff,
+  team,
   teamBuff,
 } from '../../util'
-import { entriesForChar, registerAllDmgDazeAndAnom } from '../util'
+import { entriesForChar, getBaseTag, registerAllDmgDazeAndAnom } from '../util'
 
 const key: CharacterKey = 'Soldier0Anby'
 const data_gen = allStats.char[key]
 const dm = mappedStats.char[key]
+const baseTag = getBaseTag(data_gen)
 
 const { char } = own
 
-// TODO: Add conditionals
-const { boolConditional, core_markedWithSilverStar } = allBoolConditionals(key)
-const { listConditional } = allListConditionals(key, ['val1', 'val2'])
-const { numConditional } = allNumConditionals(key, true, 0, 2)
+const { markedWithSilverStar } = allBoolConditionals(key)
 
 const sheet = register(
   key,
   // Handles base stats, core stats and Mindscapes 3 + 5
   entriesForChar(data_gen),
-  registerBuff(
```

Comment:
can you leave this one for now, we can remove it when we migrate people over to real sheets